app/controllers/ReportingController.scala

```diff
@@ -17,26 +17,28 @@
 package controllers
 
 import connectors.AuthProviderClient
+import model.ApplicationRoute
 import model.Commands._
 import model.command.ProgressResponse
 import model.persisted.ContactDetailsWithId
-import model.report.{ DiversityReportItem, OnlineTestPassMarkReportItem, ProgressStatusesReportLabels, _ }
+import model.report.{DiversityReportItem, OnlineTestPassMarkReportItem, ProgressStatusesReportLabels, _}
```

Comment:
Someone's reset their intellij and now it's doing the old { formatting again isn't it?